octodns/processor/arpa.py

```diff
@@ -79,12 +85,13 @@ def populate(self, zone, target=False, lenient=False):
         for arpa, fqdns in self._records.items():
             if arpa.endswith(f'.{zone_name}'):
                 name = arpa[:-n]
+                _, record_ttl, _ = fqdns[0]
```

Comment:
If the ttls across duplicate fqdns don't match this will pick the first one it sees which may be a little confusing to users since they won't necessarily be able to predict what will get picked without knowing the order things end up in the list. Worst case it could even varry which one was first in the list from run to run and cause ttl churn.

Reply:
Yes, true. I didn't think of the scenario where multiple PTR values exist. I use max_auto_arpa=1 in my setup. Would the other option (auto_arpa_ttl directly on the A/AAAA) make more sense then?

Reply:
Not opposed to a way to do an explicit `max_auto_arpa`, but i like the copy-over option as well. I'd probably use/prefer the copy option myself, but would accept a another PR with the explicit option if other people would use it.